src/generic/Misc/Localization.jl

```diff
@@ -113,6 +113,8 @@ base_ring(L::LocalizedEuclideanRing) = L.base_ring::base_ring_type(L)
 
 parent(a::LocalizedEuclideanRingElem) = a.parent
 
+characteristic(::LocalizedEuclideanRing) = characteristic(base_ring(L))
```

Comment:
I think we don't have any tests for `LocalizedEuclideanRing` in AA, do we? There seem to be some in Nemo, though: `test/HeckeMiscLocalization-test.jl`.

Anyway, perhaps we can add some test for it here, and perhaps even conformance tests?

Reply:
Also, this doesn't quite fit in with the title of the PR. Perhaps it should be moved into a separate PR, too?